lib/src/facts/resolvers/filesystem_resolver.cc

```diff
@@ -27,6 +31,16 @@ namespace facter { namespace facts { namespace resolvers {
     {
         auto data = collect_data(facts);
 
+        boost::program_options::variables_map vm;
+        auto hocon_conf = load_default_config_file();
+        load_fact_settings(hocon_conf, vm);
+        string mountpoint_filter = "";
```

Comment:
no need to explicitly assign empty string value, as default constructor does the same.